doc/samples/samples_list.rst

```diff
@@ -36,4 +41,4 @@ The following table demonstrates the variants' configuration for the supported d
 
 .. include:: ../ncs_links.rst
 
-.. _Semtech SX1262MB2CAS: https://www.semtech.com/products/wireless-rf/lora-transceivers/sx1262mb2cas
+.. _Semtech SX1262MB2CAS: https://www.semtech.com/products/wireless-rf/lora-transceivers/sx1262mb2casbk

```

Comment:
Should this have been updated? the new link doesn't work

Reply:
I'm not aware about any link change.
Do you know why we should change the link here?

Reply:
Got it :) This must've been a typo in this PR then, let's bring the previous link back
```suggestion
.. _Semtech SX1262MB2CAS: https://www.semtech.com/products/wireless-rf/lora-transceivers/sx1262mb2cas
```